numpy/core/src/umath/ufunc_object.c

```diff
@@ -6459,7 +6415,7 @@ static PyObject *
 ufunc_get_identity(PyUFuncObject *ufunc, void *NPY_UNUSED(ignored))
 {
     npy_bool reorderable;
-    return _get_identity(ufunc, &reorderable);
+    return PyUFunc_GetIdentity(ufunc, &reorderable);
```

Comment:
I think it would be nice to change this too, to not need the `reorderable` any more.

Reply:
Happy to change this, but as it is, if someone were to think of adding a new "default", they at least see that the two are currently linked?